contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,98 @@
+Reporting Guidelines
+====================
+
+If you believe someone is violating the code of conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** In some cases we may determine that
```

Comment:
This is highly disturbing to me --> `All reports will be kept confidential.`

I'd like to read a full explanation why that is needed at all. As I commented [here](https://github.com/symfony/diversity/issues/1#issuecomment-358565231) it's easy to be 100% transparent and at the same time, protect the privacy of anyone involved. Example of a public report:

*"A person reported that two people harassed them on Symfony Slack. We talked with person A and they admitted the wrongdoing, apologized to the victim and promised to behave in the future. Person B rejected the accusations but given the proofs in our possession, we banned them from the Symfony Slack for 3 months."*

You only talk about "a person", "person A", "person B", etc. No names, no genders, no countries of origin, no details at all. But at the same time, you offer full transparency to the community, which is much needed in these cases.

Reply:
I agree. I think we should do both. I see a value of keeping a non-anonymized report as well. That one should be confidential.

Reply:
That being said .. in some cases we will have to make the anonymized version to just read "there was an incident" in order to protect the people involved.

the idea was also raised to produce a summary report now and then (maybe every X months). in a summary it is easier to mention more details without causing issues with privacy.

Reply:
I fully disagree. No summaries please. We need full disclosure (full anonymous disclosure). The community must watch the enforcement team to ensure that they are not abusing their power.

Reply:
The stated goal of the enforcement team imho is #1 protecting people involved in incidents #2 evolving the community. As such #1 is imho more important to #2 .. if full disclosure causes issues with #1, then #2 looses. in fact I think it may at times even be necessary for the enforcement team to lie in public to ensure #1.

Reply:
> in fact I think it may at times even be necessary for the enforcement team to lie in public

In my opinion, this is a dangerous thing to say. I'd expect the enforcement team to be honest about what happened, while at the same time protecting the privacy of all parties involved. The fact that "lying to the public" is an option makes the whole thing less legitimate: how am I supposed to know that all involved parties were treated fairly and equally (and that I would be if I'd be involved in something in the future) if I don't know if you're telling the truth in a report?

Reply:
@alcaeus I totally understand the concern. I don't know how else to address situations where privacy and transparency collide. What should maybe be made clear is that in-transparency is only allowed to protect privacy and only for that exact reason.

Reply:
> What should maybe be made clear is that in-transparency is only allowed to protect privacy and only for that exact reason.

Yes, that would most likely help clarifying why the rule is in place. Thanks! 👍

Reply:
I agree that privacy needs to come before transparency. But I would prefer to just not lie in these cases, but just not disclose as many things as there are necessary to protect someone's privacy. Which at last could mean that an incident would only be included in a summary report or the like.

Reply:
lying is really just an edge case. in most cases simply omitting personal information will do the trick.

Reply:
let me give you an example:

lets say someone is leaking personal information, like postal address or information about sexual orientation etc. in such a case the enforcement team might lie by saying the leaked data is actually incorrect, even if they know them to be correct.

Reply:
There's no need to lie. An anonymous report in that case would rea like this:

-----

Person A notified us that Person B was leaking some personal information about them in different electronic mediums. The Enforcement Team found this a violation of XXXX and banned Person B for 1 year form Symfony Slack, Symfony repositories at GitHub and Symfony Conferences.

-----

As you can see, you can give a much needed transparency to the community without revealing anything or putting any person in danger.

-----

A different thing is how the team will archive this case in the private records. Will the record say: "Real Person Name B was saying that Real Person Name A was gay. We verified that doing XXX and YYY and we banned Real Person Name B for one year".

If you don't want to store those details, replace "B was saying that A is gay" by "B was leaking personal information about A".

Reply:
It might be necessary to lie in order to derail people from using the leaked information against the community member.

Reply:
I agree with Javier about transparency. I also agree with Lukas that privacy trumps transparency. We want to create a safe environment. 

I would really like us to do both as Javier suggests. I've tried to update the texts so that is more clear.